luceedebug/src/main/java/luceedebug/Agent.java

```diff
@@ -63,9 +63,11 @@ static class AgentArgs {
                         gotJdwpHost = true;
                         break;    
                     }
-                    case "cfhost": {
-                        cfHost = value;
-                        gotCfHost = true;
+                    // cfhost is deprecated; replaced with debughost

```

Comment:
Please do like
```
case foo:
  // fallthrough (... reason ...)
case bar:
 ...
 ```
to make it explicit that fallthrough is intentional.

C++ has `[[fallthrough]];` which informs the compiler that fallthrough is desired, do you know if Java has anything like that?

Reply:
FYI, there's [no java compiler annotation](https://stackoverflow.com/questions/58338640/indicating-that-a-switch-case-falls-through) for fallthrough behavior, AFAIK.

While the following might be considered technically a subcategory of fallthrough, I think that idiomatically, it's just two labels for the same case statement. Oracle describes both fallthrough and "multiple labels per case statement" [here](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/switch.html).

```java
case 'foo': case 'bar': // <-- this
  doSomething();
 break;
case '
...
```

One thing that we _could_ do would be to put them on the same line, as shown above. Maybe that would be the multi-label syntax clearer?

Let me know what you think. We can also just go with your original suggestion.

Reply:
Just the original suggestion for that, please and thankyou.

Reply:
Done.